workflows/wf_titan_illumina_se.wdl

```diff
@@ -95,10 +95,10 @@ workflow titan_illumina_se {
     String  seq_platform = seq_method
 
     File    read1_clean                 = read_QC_trim.read1_clean
-    Int     fastqc_raw                  = read_QC_trim.fastqc_number_reads
-    String  fastqc_version              = read_QC_trim.fastqc_version
+    Int     fastq_scan_raw                  = read_QC_trim.fastq_scan_number_reads
+    String  fastq_scan_version              = read_QC_trim.fastq_scan_version
```

Comment:
Same here `fastq_scan_raw_version`?

Reply:
This one uses that [subworkflow](https://github.com/theiagen/public_health_viral_genomics/blob/kgl-fastq-scan-dev/workflows/wf_read_QC_trim_se.wdl#L72) that does take the `fastq_scan_raw_version`